src/main/resources/db/migration/V001.1__CREATE_BOOKS_TABLE.sql

```diff
@@ -14,4 +14,4 @@ create table books
     isbn             varchar(255),
     primary key (id)
 );
-alter table if exists books add constraint fk_author_id foreign key (author_id) references author;
+
```

Comment:
We should not edit flyway scripts that were already executed, let's just keep things as they were